### ISSUE TITLE
Why Top Bar and Left Menu did not include a single page?

### ISSUE THREAD

**Issue by oguzhancevik** [BlackrockDigital/startbootstrap-sb-admin-2#339]
Why do we write the menu and top bar again on every page?

Constant items like menus and topbars need to be defined on a different single page.

Other content pages also call up the page containing these fixed items.

**Comment by tiesont** [comment]
This is a template. It's not intended that you use it as-is.

**Comment by oguzhancevik** [comment]
Yes you are right. But we must prevent the wrong usage. Maybe some developers code has duplicate code per page.

**Comment by oguzhancevik** [comment]
[https://stackoverflow.com/questions/18712338/make-header-and-footer-files-to-be-included-in-multiple-html-pages](url)

@davidtmiller

**Comment by davidtmiller** [comment]
I'll be moving to Pug in the near future to address this. The source files will use Pug to handle includes that can be used across all of the pages, and the rendered HTML will remain static for those who don't want to use Pug. This change is coming soon!

**Comment by oguzhancevik** [comment]
> I'll be moving to Pug in the near future to address this. The source files will use Pug to handle includes that can be used across all of the pages, and the rendered HTML will remain static for those who don't want to use Pug. This change is coming soon!

Thank you @davidtmiller
